File: Kapil RNO 39/DotNet/Assignment 3/DemoApp/Banner/Banner.cs
namespace banner;
class Banner
{
    public float width { get; set; }
    public float height { get; set; }

    public Banner()
    {
        width = 0;
        height = 0;
    }
    public Banner(float width, float height)
    {
        this.width = width;
        this.height = height;

    }

    public virtual double Area()
    {
        return width * height;
    }

}

File: Kapil RNO 39/Java/Assign4/src/Banner.java
package banner;

public class Banner {
	
	private float width;
	private float height;

	public Banner(float w, float h) {

		width = w;
		height = h;
	}

	public float getWidth() {
		return width;
	}

	public float getHeight() {
		return height;
	}

	public void setWidth(float w) {
		width = w;
	}

	public void setHeight(float h) {
		height = h;
	}

	public double area() {

		return width * height;
	}

}
	
